### PR TITLE
[CI] Swagger docs build failing

### LINKED ISSUE #8513: [CI] Swagger docs build failing
### Current Behavior
Please see https://github.com/meshery/meshery/actions/runs/5896636109/job/15994835850

### Acceptance Tests

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)
- 🧪 [Meshery Test Plan Spreadsheet](https://docs.google.com/spreadsheets/d/13Ir4gfaKoAX9r8qYjAFFl_U9ntke4X5ndREY1T7bnVs/edit#gid=0)

### PR DESCRIPTION
**Pull request by suhail34**
**Notes for Reviewers**

This PR fixes #8513 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by suhail34** [comment]
I have to check what responses they written and edit it accordingly

File: server/handlers/database_handlers.go [hunk 1]
```diff
@@ -116,20 +116,10 @@ func (h *Handler) GetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *m
...
-// A successful response indicating that the database reset operation was successful.
-// ---
-// type: string
-// example: "Database reset successful"
-
-// errorResponse: error response
-// An error response indicating the reason for the failure of the database reset operation.
-// ---
-// type: string
-// example: "Directory could not be created due to a non-existent path."
```
Comment (leecalcote) [inline server/handlers/database_handlers.go:132]: @Aritra8438 is this accurate?

Comment (Aritra8438) [reply server/handlers/database_handlers.go:132]: Yes @leecalcote. There might be many reasons that the databse reset is not successful. 
"Directory could not be created due to a non-existent path." is one of the possibilities.

Comment (Aritra8438) [reply server/handlers/database_handlers.go:132]: Hi, @leecalcote, I think the problem is in `200: successfulResponse` considering the issue you mentioned.

I've already mentioned the successfulResponse after this, but that should be mentioned at `handlers.go`.

Should I link a more definite PR?

**Comment by leecalcote** [comment]
Thanks for confirming 👍

**Pull request merged**